src/js/components/SearchInput/SearchInput.tsx

```diff
@@ -1,17 +1,16 @@
-// @flow
 import * as React from 'react';
 import classNames from 'classnames';
-import { FormInput, Button } from '../../';
+import { FormInput, Button } from '../..';
 
-type Props = {
-  className?: string,
-  onChange: (SyntheticKeyboardEvent<HTMLInputElement>) => void,
-  onSearch: (SyntheticKeyboardEvent<HTMLInputElement> | SyntheticMouseEvent<>) => void,
-  placeholder?: string,
-  value: string,
-};
+interface Props {
```

Comment:
Is there a reason why you used `interface` here instead of `type`?

Reply:
I was extending to get inputs attributes previously so I needed to use an interface. I'll change back to type